app/services/api/webflow-api.service.ts

```diff
@@ -0,0 +1,41 @@
+import { AxiosInstance } from "axios"
+import {
+  ListCollectionsResponseSchema,
+  GetCollectionResponseSchema,
+  GetCollectionItemResponseSchema,
+} from "./webflow-api.schema"
+import { z } from "zod"
+
+export const Webflow = (deps: { api: AxiosInstance }) => ({
```

Comment:
Created this service so we could use it in query functions and `scripts/webflow-codegen.ts`